README.md

```diff
@@ -1,9 +1,62 @@
-# Parity GitHub Action template
+# GitHub metrics
 
-Template used to generate GitHub Actions.
+Get GitHub metrics for a repository.
+
+Supports obtaining metrics from:
+
+- Pull Requests
 
 ## To start
 
-- Remember to modify the `action.yml` file to have your required attributes and details. 
- - You can use [GitHub Action Brandings cheatsheet](https://github.com/haya14busa/github-action-brandings) to set the style of the action.
-- Remember to modify the name in the `package.json`.
+### Run from CLI
+
+Copy `example.env` as `.env` and replace the variables with your own.
+
+Run:
+
+- `yarn install`
+- `yarn run cli`
+
+It will generate a file named `report.md` as the output.
+
+### Run as GitHub action
+
+Install this action in your repository:
+
+```yml
+name: Fetch metrics
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 12 1 * *"
+
+jobs:
+  get_metrics:
+    runs-on: ubuntu-latest
+    name: Get repository metrics
+    steps:
+      - uses: paritytech/metrics@main
+        name: Fetch polkadot-fellows/runtimes metrics
```

Comment:
I know this is just an example, but it is named about `polkadot-fellows/runtimes` metrics but gets `polkadot-fellows/metrics` metrics so it's confusing.

Reply:
I removed it so I don't get confused again